### ISSUE TITLE
error at line 14891

### ISSUE THREAD

**Issue by asdfasdffd** [PoE-TradeMacro/POE-TradeMacro#833]
says- Line Text: C:\Users\****\AppData\Local\Temp 
Error: this line does not contain a recognized action
program will now exit.

**Comment by Remo888** [comment]
I am getting the same issue. i have tried unzipping with winrar and winzip both gave the same result. still cant run the macro

**Comment by Eruyome** [comment]
Where have you unzipped the folder to?

**Comment by asdfasdffd** [comment]
Well ive tried having the folder in my D drive and also in my downloads on my C drive neither is working

**Comment by scrusi** [comment]
Have a similar issue (same line, different error text) with the no-exe version. Definitely unzipped, tried different folders and drives, swapped AHK 1.1.30 to 1.1.29, nothing helps. Used to run fine until a recent Windows re-install. Looks like either an encoding error or trying to read a binary to me (see screenshot).

![tm_error](https://user-images.githubusercontent.com/137942/50178098-e992ad80-0303-11e9-886c-7952fbc4bdfc.JPG)

**Comment by Remo888** [comment]
yes this started happening to mine aswell after a recent windows reinstall, prior to reinstall TM was working perfect for me

**Comment by Eruyome** [comment]
Are you aware of any recent windows update (which would be installed when reinstalling windows) that could have caused this?

I can't find the ticket right now but if I remember correctly there's at least one person where this or a similiar error occured after not playing for a while, in the meantime the only thing that changed where windows 10 updates.

**Comment by Eruyome** [comment]
In your case (@scrusi ) this happen when the file `%userprofile%\Documents\PoE-TradeMacro\CustomMacros\customMacros_example.txt` is being included in the script.

Please check whether this file is broken in any way (contents, encoding).

**Comment by scrusi** [comment]
Ah, that file must have been corrupted at some point. (I rescued the Documents folder from the old hard drive, the corruption of which was the cause for the re-install.) Renaming the file fixed the issue. I've attached the old file in case you need it for trouble shooting.
[customMacros_example_broken.txt](https://github.com/PoE-TradeMacro/POE-TradeMacro/files/2694485/customMacros_example_broken.txt)

Thank you for your help and your good work!

**Comment by Eruyome** [comment]
I guess I should find a way to check whether the file is OK before actually trying to merge it into the script. It's easy to just skip the file while warning users about it.

**Comment by asdfasdffd** [comment]
![broke](https://user-images.githubusercontent.com/45705057/50263415-9c860880-03e3-11e9-8f57-b1b1ec100193.png)
So no luck with mine? I had sent the picture in a private message before i think my bad.

**Comment by Eruyome** [comment]
Same mentioned line but different error message... did you also try deleting the file that I mentioned?

**Comment by Eruyome** [comment]
I need the file from a time where it didn't work.

**Comment by asdfasdffd** [comment]
[_TradeMacroMain.zip](https://github.com/PoE-TradeMacro/POE-TradeMacro/files/2704908/_TradeMacroMain.zip)
ok the program throws the error in this one

**Comment by tobiaswi** [comment]
hey @Eruyome i have the same issue. it started today and i updated windows yesterday.
my error is this:
![image](https://user-images.githubusercontent.com/15244405/50376429-03d2d100-060d-11e9-96c0-4e04ed1a013f.png)

the windows updates are:
![image](https://user-images.githubusercontent.com/15244405/50376434-13eab080-060d-11e9-8ad2-eabae84608b1.png)

can i do something to help`?

**Comment by lucasrodc** [comment]
Found a solution to my problem.

Go to the documents folder / and delete everything about the trade macro, and try opening the trademacro again
